docs/blog/2025-06-12-nx-21-2-release.md

```diff
@@ -0,0 +1,78 @@
+
+## What's new in Angular 20
+
+We'll leave it to the Angular team for [all the details](https://blog.angular.dev/announcing-angular-v20-b5c9c06cf301), but we wanted to call out a few changes that may impact Nx workspaces.
+
+### Structural directives deprecated
+
+It's time to part ways with structural directives like `*ngIf` and `*ngFor` and move on to control flow with @if and @for.
+
+Your workspace will be migrated to control flow by default during migration to Nx 21.2. If you’d like to hang on to that syntax a little longer, see our docs on [running migration in interactive mode](/recipes/tips-n-tricks/advanced-update) to skip this migration. Or use the newly introduced [Migrate UI in Nx Console](/recipes/nx-console/console-migrate-ui) to skip the migration.
+
+If you choose not to migrate now, you’ll be able to re-run the migration later:
+
+```shell
+nx g @angular/core control-flow
+```
+
+### Experimental Support for vitest from the Angular team
+
+Angular 20 introduces experimental support for [vitest](https://vitest.dev/) as a unit test runner. This is a fantastic addition to the core set of tools offered by the Angular team. Nx already offers vitest as an option when creating Angular apps and libraries, which is enabled by installing the [Analog plugin](https://analogjs.org/docs/features/testing/overview). You can choose between vitest support via the Analog plugin or through the new experimental vitest support. Nx generators will continue to use the Analog plugin, see the [Angular documentation](https://angular.dev/guide/testing/unit-tests#) on how to use the vitest builder.
+
+### New style guide changes
+
+Last year, the Angular team opened [an RFC for changes to the style guide](https://github.com/angular/angular/discussions/58412), and Angular 20 implements the changes discussed. One of the bigger changes is that Angular CLI will not add type suffixes to filenames for your components, directives, services, and pipes by default. So rather than generating a component in a file named `button.component.ts`, it now generates `button.ts`. Generators provided by Nx will also follow the new style guide for new workspaces using Angular, but **the previous behavior will be retained for existing workspaces**. This is the same process the Angular CLI is following.
+
+As part of migrating to Nx 21.2, generator and schematic defaults will be set for your workspace so that filename suffixes continue to be added. You can see what that change looks like in the [migration documentation](/technologies/angular/api/migrations#setgeneratordefaultsforpreviousstyleguide).
+
+### BREAKING CHANGE: Angular 17 support deprecated
+
+Following our [support schedule](/technologies/angular/recipes/angular-nx-version-matrix), Angular 17 is no longer supported in Nx 21.2. If you want to continue with Angular 17, you'll need to postpone upgrading to Nx 21.2 for now.
+
+![NestJS logo on a dark background](/blog/images/2025-06-12/nest.avif)
```

Comment:
should this be within the `NextJS 11 is here` section? 🤔 Honest question, intuitively it felt like weird at a first impression seeing the NestJS logo within the Angular deprecated section. At a 2nd glance I understood you moved to having it before the section rather than within..